### ISSUE TITLE
Piping command output to clipboard

### ISSUE THREAD

**Issue by reagle** [hluk/CopyQ#1057]
Shouldn't the following work?

```
─➤  echo "Some text" | copyq copy
Note: Started User activity for: CopyQ Client
Note: Ended activity
DEBUG: Message received: CommandException
ScriptError: Failed to copy to clipboard!
```

- copyq: 3.7.1
- MacOS Mojave

**Comment by reagle** [comment]
Maybe we could show this in the examples.

**Comment by hluk** [comment]
Not exactly the same command, but `copyq copy - < file.txt` is mentioned on the [web site](https://hluk.github.io/CopyQ/) in the README file.

I accept pull requests so feel free to update it (or the documentation).

**Comment by reagle** [comment]
That's true. In my mind I was thinking of pbcopy, which doesn't require the `-`.
